--- lib/python2.7/site-packages/eztables/views.py.ori	2014-04-28 15:11:17.346291283 +0200
+++ lib/python2.7/site-packages/eztables/views.py	2014-04-28 15:11:29.173043113 +0200
@@ -58,9 +58,11 @@
     _db_fields = None
 
     def post(self, request, *args, **kwargs):
+        self.POST = request.POST
         return self.process_dt_response(request.POST)
 
     def get(self, request, *args, **kwargs):
+        self.GET = request.GET
         return self.process_dt_response(request.GET)
 
     def process_dt_response(self, data):
